### ISSUE TITLE
Determine if bigdecimal is an integer

### ISSUE THREAD

**Issue by sunjay** [akubera/bigdecimal-rs#46]
It would great if there was a way to determine if the `BigDecimal` was an integer without converting it to `i64`. The check would need to determine if there is anything on the right of the decimal point.

```rust
impl BigDecimal {
    pub fn is_integer(&self) -> bool {
        //...
    }
}
```

**Comment by akubera** [comment]
Finally, this has been merged and pushed into 0.1.0. enjoy

**Comment by sunjay** [comment]
Thank you!
